apis/scheduling/v1alpha1/reservation_types.go

```diff
@@ -32,20 +32,26 @@ type ReservationSpec struct {
 	// like a normal pod.
 	// If the `template.spec.nodeName` is specified, the scheduler will not choose another node but reserve resources on
 	// the specified node.
+	// +optional
 	Template *corev1.PodTemplateSpec `json:"template,omitempty"`
 	// Specify the owners who can allocate the reserved resources.
-	// Multiple owner selectors and ANDed.
+	// Multiple owner selectors and ORed.
+	// +optional
 	Owners []ReservationOwner `json:"owners,omitempty"`
 	// By default, the resources requirements of reservation (specified in `template.spec`) is filtered by whether the
 	// node has sufficient free resources (i.e. ReservationRequest <  NodeFree).
 	// When `preAllocation` is set, the scheduler will skip this validation and allow overcommitment. The scheduled
 	// reservation would be waiting to be available until free resources are sufficient.
+	// +optional
 	PreAllocation bool `json:"preAllocation,omitempty"`
 	// Time-to-Live period for the reservation.
 	// `expires` and `ttl` are mutually exclusive. Defaults to 24h.
+	// +kubebuilder:default='24h'
```

Comment:
It is recommended to turn the changes of the Reservation CRD into a separate PR.

Reply:
it does not matter. It is acceptable to change this CRD definition at this stage.